test_scripts/Protocol/NACKReason/004_Invalid_Certificate_NACK.lua

```diff
@@ -0,0 +1,103 @@
+---------------------------------------------------------------------------------------------------
+--  Precondition:
+--  1) Initialize the client side certifcate file for SDL
+--  2) Start SDL, HMI, connect Mobile device
+--  3) Register App_1 and App_2 on SDL.
+--
+--  Steps:
```

Comment:
description here doesn't match the test

Reply:
Good catch. Fixed in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2436/commits/5dd8216630c3968e4dc114a1059340f75dd157de